wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -500,15 +500,20 @@ extension ConversationCreationController {
     }
 }
 
-extension Feature.MLS.Config.MessageProtocol {
-    var toMessageProtocol: MessageProtocol {
-        switch self {
+extension UserSession {
+
+    var defaultProtocol: MessageProtocol {
+        guard mlsFeature.isEnabled else {
+            return .proteus
+        }
+        switch mlsFeature.config.defaultProtocol {
```

Comment:
I think here you can remove the switch and directly return `mlsFeature.config.defaultProtocol.toMessageProtocol`

Reply:
Unfortunately, I can't. 
mlsFeature.config.defaultProtocol is `Feature.MLS.Config.MessageProtocol `type. But I would like to have a result as `MessageProtocol` enum type.

Reply:
that's not the case here ?

```
extension Feature.MLS.Config.MessageProtocol {
    var toMessageProtocol: MessageProtocol {
        switch self {
        case .proteus:
            .proteus
        case .mls:
            .mls
        case .mixed:
            .mixed
        }
    }
}
```